### ISSUE TITLE
LateJoin does not work as intend.

### ISSUE THREAD

**Issue by tiradorus** [Eredrim/pvparena#63]
Hey! I have a bug with your last version of PvPArena (pvparena-1.15.1-SNAPSHOT-b350)

When an arena is runing, with people inside, then players that are late and came after the first start of the game cannot enter into the arena.
Everything is allright into the config, it's not a problem.

When they do a "**/pa (Map) join**" they are teleported into the TeamLounge, they can click onto the iron-block to be ready, it put them into a team but ... nothing happen then.
They are still locked onto the lounge of their team. They are not teleported into the game. Furthermore, they are immortal and cannot use their items. (As if they still were into the lounge, even if they have said that they were ready)

The only way to make them join the game lately, is to kill them by a /kill or a /suicide. AND ONLY THEN: they can play the game with other players.
Thanks to fix it! =)

Minecraft version: 1.16.2 (paper)

**Comment by Eredrim** [comment]
Hello,

Thanks for reporting! I have two questions in order to focus the problem:
- What did you try to do? Allowing player to join an arena during a match or blocking this possibility?
- Did you enable `joinInBattle` and/or `alwaysJoinInBattle` in your arena? Without these parameters, joining a running arena is not permitted.

**Comment by tiradorus** [comment]
As i said above:
"Everything is allright into the config, it's not a problem."

I've tryed everything. JoinInBattle: true with AlwaysJoinInBattle: true; everything on false; one true the other false ... 
I've tryed everything so  ^^'
I think, for what i want to do is to have both on true

I just want to allow players joining an arena in every time. Even if a game is already launched on this arena. To join the actual battle even if the player is late.

I have Events on my server that always start at 19h.00, however sometimes a player is connecting a 19h01, he is late but not that much, and i want to allow him to join the arena even if he was a bit late.
Or even more, sometimes a player have a crash during the game, and he wanna come-back. I want to allow that.
However, with the current bug, he can came-back into the lounge, click on the iron-block, the plugin says that he is ready and ... and can't leave the lounge... He is stuck there. Nothing happen. He is not teleported into the battlefield
I have to kill the player to make him joining the game (With a /kill beacause he is invincible)
